kubernetes-deployment/test/data/data_maintainlib_common/repair-worker-node.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
```

Comment:
are this file and kubelet.sh.template the same as those in kubenetes-deployment. if yes, maybe we can copy the file here before running the test. (e.g., can we put the file copy code in kubernetes-deployment/test/__init__.py ?)

Reply:
Don't do too much. Just copy it here, this is a better way.